### DISCUSSION TITLE
Is there a way to actually sign in?

### DISCUSSION THREAD

**Discussion by iamqw3rty** [YoungFellow-le/photoshop-22-linux#23] [Ideas]
Unlike most other people (joke) I actually pay for Photoshop, so I would love it if I could actually sign in, as I want to be able to save stuff to the cloud and use the fonts I get from Adobe.

**Comment by albakhtari** [comment]
Apparently not, the views in which you sign in won't load under `wine`, they are the same as the installer view.
I have Photoshop on a Windows VM to use for all the things I can't do using PS running on wine.
